tools/metadata_viewer/kafka.py

```diff
@@ -0,0 +1,13 @@
+from storage import Segment
+
+
+class KafkaLog:
+    def __init__(self, ntp):
+        self.ntp = ntp
+        self.batches = []
+
+    def decode(self):
+        for path in self.ntp.segments:
+            s = Segment(path)
+            for batch in s:
+                self.batches.append(batch.header._asdict())
```

Comment:
maybe it would be better to use python generator, it may be hard to gather all batches in memory